docker-compose.yaml

```
@@ -1,35 +1,17 @@
-  # db:
-  #   image : infront/mssqldev:2019
-  #   build : ./etc/dbext
-  #   environment:
-  #     - ACCEPT_EULA=Y
-  #     - SA_PASSWORD=Admin@123456
-  #   ports:
-  #     - 1433:1433
-
-
-    
+      - 127.0.0.1:8000:8000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)